### ISSUE TITLE
Adjust Training Process

### ISSUE THREAD

**Issue by sagedemage** [sagedemage/pet_image_classification_system#15]
**Describe the enhancement**
Adjust the *epochs*, and *learning rate* parameters of the training process to ensure that the **validation loss** decreases more and to improve the **convergence speed** of the validation loss. This could also improve the accuracy of the model.

**Comment by sagedemage** [comment]
```
git fetch origin
git checkout 15-adjust-training-process
```
